.test-infra/jenkins/jira_utils/jira_manager.py

```diff
@@ -199,20 +211,63 @@ def _find_owners(self, dep_name):
       owners = dep_info['owners']
       if not owners:
         logging.warning("Could not find owners for " + dep_name)
-        return None, []
+        return []
     except KeyError:
       traceback.print_exc()
       logging.warning("Could not find the dependency info of {0} in the OWNERS configurations.".format(dep_name))
-      return None, []
+      return []
     except Exception as e:
       traceback.print_exc()
       logging.error("Failed finding dependency owners: "+ str(e))
-      return None, None
+      return None
 
     logging.info("Found owners of {0}: {1}".format(dep_name, owners))
     owners = owners.split(',')
     owners = map(str.strip, owners)
     owners = list(filter(None, owners))
-    primary = owners[0]
-    del owners[0]
-    return primary, owners
+    return owners
+
+
+  def _need_reopen(self, issue, dep_latest_version):
+    """
+    Return a boolean that indicates whether reopen the closed issue.
+    """
+    # Check if the issue was closed with a "fix version/s"
+    # Reopen the issue if it hits the next release version.
+    next_release_version = self._get_next_release_version()
+    if issue.fields.fixVersions[0].name in next_release_version:
```

Comment:
Should we compare against all fix versions ?

Reply:
Nice catch. Done.